content/docs/0.14.x/bridge/manage_sequences/index.md

```diff
@@ -65,13 +65,20 @@ Fields:
 link="./assets/trigger-05-trigger-evaluation-timeframe"
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation will be performed to the past for the given timeframe. This is the only case where evaluations will be performed to the past.
+* As soon as a start date is given, the evaluation is always performed from this point to an end date. The end date is either given or will be calculated from the timeframe.
```

Comment:
When a start date is specified, the evaluation is perfromed from this point to an end date.  The end date can be specified or is calculated from the timeframe value.